doc/scripts/GMT_earthmask.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+#
+# Plot the earth masks
+
+gmt begin GMT_earthmask ps
+	gmt makecpt -Clightblue,burlywood,blue,red,white -T-0.5/4.5/1 -N
+	gmt grdimage @earth_mask_05m_p.grd -JQ0/15c -Baf -BWsNe
+	# BUG IN MASK. The next call covered it up for the orig so it will fail until we fix the mask
+	#echo -40 -82 | gmt plot -Ss0.3c -Gdarkbrown
```

Comment:
Is the comment still valid? What does it mean?

Reply:
Obsolete, now removed.